### ISSUE TITLE
private-methods loose doesn't work without class properties

### ISSUE THREAD

**Issue by nicolo-ribaudo** [babel/babel#9424]
## Bug Report

**Input Code**

```js
  class X {
    #foo() {}
  }
```

**Current output**

```js
class X {
  constructor() {
    _foo.add(this);
  }

}

var _foo = new WeakSet();

var _foo2 = function _foo2() {};

```

**Expected output**
```js
class X {
  constructor() {
    Object.defineProperty(this, _foo, {
      value: _foo2
    });
  }

}

var _foo = babelHelpers.classPrivateFieldLooseKey("foo");

var _foo2 = function _foo2() {};

```

**Babel Configuration (.babelrc, package.json, cli command)**

```json
{
  "plugins": [
    ["external-helpers", { "helperVersion": "7.1000.0" }],
    ["proposal-private-methods", { "loose": true }]
  ]
}

```

**Environment**
- Babel version(s): v7.3.0

**Comment by nicolo-ribaudo** [comment]
If anyone wants to take a look at this issue, the problem is probably in the entry point of `@babel/helper-create-class-features-plugin`.

**Comment by Chrisyee22** [comment]
Could I have a look at this issue?

**Comment by pnowak** [comment]
Hi @Chrisyee22  Can I take over the issue? Please let me know :)

**Comment by nicolo-ribaudo** [comment]
@pnowak If Chris doesn't respond in ~2 days, feel free to work on this bug!

**Comment by Chrisyee22** [comment]
@pnowak go ahead. It’s all yours

**Comment by nicolo-ribaudo** [comment]
Fixed (3aaafae053fa75febb3aa45d45b6f00646e30ba4)
